### PR TITLE
Awesome-Book works

### PR DESCRIPTION
**Pull request by melkamu12**
-  Used es6 syntax.
-  Used import and export methods.
- used Luxon for a date and time.

### CODE REVIEW

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @melkamu12 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not use the correct gitflow for this project please do that and try again 👍🏾 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by melkamu12** [comment]
- It is global Error it is byond my skill to fix.. i see the slack and they put all message like this on the linters error. They got approvement with this way.

**Review by DammyShittu — COMMENTED** [review]
Hi @melkamu12,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the microverse logo from your readme doc. Kindly remove the section descriptions. Kindly fix your live demo link, it takes the user to the google homepage. If you have no demo link kindly remove the section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by DammyShittu — COMMENTED** [review]
Hi @melkamu12,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove this part of the readme also since it is a dummy text.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -45,247 +6,153 @@ For more information on the importance of a professional README for your reposit
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (DammyShittu) [inline README.md:162]: - [ ] Kindly fix your MIT link. It is broken.

  image

File: modules/setup.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export default function setup() {
```
Comment (DammyShittu) [inline modules/setup.js]: - [ ] Kindly use an arrow function for ES6 syntax rule.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @melkamu12,

## Highlights
✔️ Arrow functions

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @melkamu12👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**